tuf-design.md

```diff
@@ -4,6 +4,36 @@ Based on discussions with the Notary v2 community over the past few months, we d
 
 This design will be used in conjunction with the OCI artifact work done by others in the Notary v2 community to ensure compliance with the OCI specification and integration with existing registries.
 
+## Alignment with Notary v2 project goals
+
+The use of TUF in Notary v2 is part of an effort to address the [goals](https://github.com/notaryproject/requirements#goals) of the Notary v2 project.
```

Comment:
Could we also address the requirements that have been agreed upon in key management?

Reply:
Added!